### PR TITLE
WIP: Waktu sesi

### PR DESCRIPTION
**Pull request by arifpriadi**
Issue https://github.com/OpenSID/OpenSID/issues/4621

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -383,6 +385,29 @@ $(document).ready(function()
 
 });
 
+function sesi_login() {
+	var sessionTimeout;
+
+	function resetSessionTimeout() {
+		console.log('reset')
```
Comment (totoprayogo1916) [inline assets/js/script.js]: hapus `console.log()`

Comment (arifpriadi) [reply assets/js/script.js]: sudah disesuaikan

**Comment by andifahruddinakas** [comment]
mas @arifpriadi, mohon di resolve conflict terlebih dahulu.

**Comment by arifpriadi** [comment]
> mas @arifpriadi, mohon di resolve conflict terlebih dahulu.

Sudah disesuaikan.

**Pull request closed without merging**